code/modules/mob/new_player/new_player.dm

```diff
@@ -379,6 +382,17 @@
 	popup.set_content(dat)
 	popup.open(FALSE)
 
+/mob/new_player/proc/ViewLore()
+	var/output = "<div align='center'>"
+	output += "<i>The <b>TerraGov Marine Corps'</b> mission is to enforce space law for the purpose of defending Terra's orbit as well as other solar colonies around the galaxy under the conflict of the Independent Colonial Confederation and the intelligent xenomorph threat. \nThe TGMC is composed by willing men and women from all kinds of social strata, hailing from all across the TerraGov systems. \nAs the vessel approaches to the ordered location on space, the cryostasis pods deactivate and awake you from your long-term stasis. Knowing that it's one of those days again, you hope that you'll make this out alive...</i><br><br>"
+
+	output += "<i>Hailing from one of Saturn's colonies, as well as Mars and other unlisted habitats, the <b>xenomorph threat</b> remains at large. Extremely dangerous extraterrestrial lifeforms, part of the hive under the Queen Mother, had caught the TGMC and NT colonies off-guard during their discovery in 2414. \nThey are divided into castes, each with their specialized roles equivalent to a traditional squad member in a human force, thanks to the xenomorph's lifecycle. \nAfter days of ravaging the current area, a metal hive was sighted by Queen Mother and spawned you there. With your intent to spread the hive is in motion, you and your fellow sisters get to work...</i>"
```

Comment:
Maybe add this part next to "Join as a Xeno" and the other next to "Join the TGMC" ? Differentiating the backgrounds/stories?

Reply:
Like make it side-by-side?

Reply:
Tried a different solution this time, as per Spyro's request.